web_src/js/features/repo-migrate.js

```diff
@@ -58,10 +59,8 @@ export function initRepoMigrationStatusChecker() {
 }
 
 async function doMigrationRetry(e) {
-  await fetch($(e.target).attr('data-migrating-task-retry-url'), {
-    method: 'post',
+  await POST($(e.target).attr('data-migrating-task-retry-url'), {
     headers: {
-      'X-Csrf-Token': csrfToken,
       'Content-Type': 'application/json',
     },
   });
```

Comment:
```suggestion
```
Sorry I had to split up the suggestion.
Thank GitHub for that.

Reply:
Done in https://github.com/go-gitea/gitea/pull/27026/commits/d4d82aa3b76d7f3cb917f7cc64fde6b9307ca83c.